<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="YP1001035DT" name="com.yash.bartersuite.Testcase.Login.Testcase_ValidLogin" tests="1" failures="0" timestamp="8 Jun 2016 10:05:38 GMT" time="1.010" errors="1">
  <testcase name="Validlogin" time="1.010" classname="com.yash.bartersuite.Testcase.Login.Testcase_ValidLogin">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.yash.bartersuite.POM.login.PageObjects_Login.txt_username(PageObjects_Login.java:23)
	at com.yash.bartersuite.TestAction.login.Action_ValidLogin.loginvaliddata(Action_ValidLogin.java:32)
	at com.yash.bartersuite.TestAction.login.Action_ValidLogin.ValidLogin_Actions(Action_ValidLogin.java:21)
	at com.yash.bartersuite.Testcase.Login.Testcase_ValidLogin.Validlogin(Testcase_ValidLogin.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at com.yash.bartersuite.CustomProperties.XML.XMLRead(XML.java:317)
	at com.yash.bartersuite.CustomProperties.Main.main(Main.java:13)
]]>
    </error>
  </testcase> <!-- Validlogin -->
</testsuite> <!-- com.yash.bartersuite.Testcase.Login.Testcase_ValidLogin -->
